web/src/components/index/indexIntro.js

```diff
@@ -12,12 +12,13 @@ const Section = styled.section`
 `;
 
 const AnimationWrapper = styled.div`
+    ${tw`w-10/12`}
```

Comment:
Add a tw breakpoint so that the animation is the full width on small screens

Reply:
@asasmith I sent you the preview, let me know if this is what you were expecting.